src/lib/ndpi_content_match.c.inc

```diff
@@ -21,7 +21,7 @@
 /* ****************************************************** */
 
 static ndpi_network host_protocol_list[] = {
-  /*
+ /*
```

Comment:
@saravananfff is there a plan to make this dynamically loadable?

Reply:
@sunilfff Not now, If we are not able add the API based apps directly into its nDPI data structure, then we might consider making this dynamically loadable and use it for updating those API based apps periodically.

Reply:
@saravananfff could you please explain, what do you mean by _If we are not able add the API based apps directly into its nDPI data structure_ ?

Reply:
I need to explore other options like adding/updating an application signature(ip/url) from click through an function call to ndpi library.
If we are able to achieve that then for new apps which has dynamic signature (identified by API's provided) can be updated through that function call itself. Otherwise we need to make this `host_protocol_list` dynamically loadable to add/update the new application signatures.